load/peering.go

```diff
@@ -20,19 +20,36 @@ const (
 	acceptorPeerName = "acceptor"
 )
 
-// KVConfig is the configuration for the KV load generator
-type PeeringConfig struct {
-	// UpdateRate time in seconds between peered service registrations
-	UpdateRate rate.Limit
+var defaultExportTimeout = time.Duration(180 * time.Second)
 
+// PeeringConfig is the configuration for the Peering load generator
+type PeeringConfig struct {
+	// RegisterLimit is the time in seconds to wait between registering services
+	RegisterLimit rate.Limit
+	// ExportTimeout is the maximum time to wait for services to become exported to a remote peer
+	ExportTimeout time.Duration
+	// PeeringClusters holds the addresses for the servers that will be peered
 	PeeringClusters []string
-
+	// NumServices is the number of services to register and export during the load test
 	NumServices int
 }
 
-// peeringCluster represents a peering connection from the
-//
-//	acceptor to the peer cluster.
+// Normalize validates the Peering load test configuration. RegisterLimit and
+// NumServices must not be zero, otherwise, an error is returned.
+func (c *PeeringConfig) Normalize() error {
+	if c.RegisterLimit == 0.0 {
+		return fmt.Errorf("invalid RegisterLimit configuration: %v", c.RegisterLimit)
+	}
+	if c.NumServices == 0 {
```

Comment:
Actually `NumServices` could be 0 since the load test can be terminited by `-timeout`.

Reply:
Got it. I'll update and add config for it.